build-tools/android-toolchain/android-toolchain.targets

```diff
@@ -36,7 +36,7 @@
   <Target Name="_UnzipFiles"
       DependsOnTargets="_DetermineItems"
       Inputs="@(_PlatformAndroidSdkItem->'$(AndroidToolchainCacheDirectory)\%(Identity)')"
-      Outputs="$(AndroidToolchainDirectory)\.stamp-sdk">
+      Outputs="@(_PlatformAndroidSdkItem->'$(AndroidToolchainDirectory)\sdk\%(DestDir)\.stamp-sdk') $(AndroidToolchainDirectory)\ndk\.stamp-ndk">
```

Comment:
if %(DestDir) is empty then the path will just be one level up, but I see your point and it makes sense.